internal/importers/base/importer_test.go

```diff
@@ -14,7 +15,17 @@ import (
 	"github.com/golang/dep/internal/test"
 )
 
+func parallelAllowed() bool {
+	// Disable parallel tests on Windows until we can resolve this error:
+	// "remote repository at https://github.com/carolynvs/deptest-importers does not exist, or is inaccessible"
+	return runtime.GOOS != "windows"
```

Comment:
I am not sure if I am comfortable hiding what is likely a bug in our code by not running parallel tests on Windows.  But it really improves our test execution time. Would you please open an issue for this error, so that we at least remember that it is a potential bug, and reference it in this function?